Sources/CombineSchedulers/TestScheduler.swift

```diff
@@ -148,20 +158,6 @@
       }
     }
 
-    /// Advances the scheduler to the given instant.
-    ///
-    /// - Parameter instant: An instant in time to advance to.
-    public func advance(to instant: SchedulerTimeType) {
-      self.advance(by: self.now.distance(to: instant))
-    }
-
-    /// Advances the scheduler to the given instant.
-    ///
-    /// - Parameter instant: An instant in time to advance to.
-    public func advance(to instant: SchedulerTimeType) async {
-      await self.advance(by: self.now.distance(to: instant))
-    }
```

Comment:
These aren't actually deleted. I just renamed `instant` to `deadline` and so the diff looks a little weird.